README.md

```diff
@@ -1,21 +1,29 @@
 # Relayer
-
-![GOZ](./docs/images/github-repo-banner.png)
+![Relayer](./docs/images/github-repo-banner.gif)
 
 ![Relayer Build](https://github.com/cosmos/relayer/workflows/Build%20then%20run%20CI%20Chains/badge.svg)
 
 The Cosmos IBC `relayer` package contains a basic relayer implementation that is
 meant for users wishing to relay packets/data between sets of IBC enabled chains.
-In addition, it is well documented and intended as an example where anyone who is
-interested in building their own relayer can come for complete, working, examples.
+In addition, it is intended as an example where anyone who is interested in building 
+their own relayer can come for complete, working, examples.
+
+**NOTE:** The relayer is in alpha and is not production ready. If it is used in production, 
+it should always be run in a secure environment and only with just enough funds to 
+relay transactions. Security critical operations **should** manually verify that the
+client identifier used in the configuration file corresponds to the correct initial 
+consensus state of the counterparty chain. This can be done by querying the initial 
+consensus state and the header of the counterparty and verifying that the root and
+hash of the next validator set match. This can be considered equivalent to checking
+the sha hash of a download or a GPG signature. 
 
 ### Security Notice
 
 If you would like to report a security critical bug related to the relayer repo, please send an email to [`security@cosmosnetwork.dev`](mailto:security@cosmosnetwork.dev)
 
 ## Code of Conduct
 
-The iqlusion team is dedicated to providing an inclusive and harassment free experience for contributors. Please visit [Code of Conduct](CODE_OF_CONDUCT.md) for more information.
+The cosmos organization is dedicated to providing an inclusive and harassment free experience for contributors. Please visit [Code of Conduct](CODE_OF_CONDUCT.md) for more information.
```

Comment:
not sure what the preferred wording is here. Maybe `cosmos community`?

Reply:
Cosmos should be capitalised. I think "Cosmos community" is fine.